libraries/ashrae/guideline36/4.9-exhaust-fan.yml

```diff
@@ -2,16 +2,21 @@ constant-speed-exhaust-fan:
   body: >
     @prefix p: <urn:___param___#> .
     @prefix brick: <https://brickschema.org/schema/Brick#> .
-    p:name a brick:Exhaust_Fan ;
-      brick:hasPoint p:start_stop, p:status, p:zat .
-  optional: ["zat"]
+    p:name a brick:Fan ;
```

Comment:
I had to make the object here a `brick:Fan` class instead of `p:name a p:constant-speed-fan` (from `components.yml` like the TTL file) because it failed `test_library.py`. There are a couple/few cases where this may be convenient and might be worth revisiting during a future template discussion/refactor. 

```bash
cls = <class 'buildingmotif.dataclasses.library._template_dependency'>, d = {'args': {'name': 'cav-fan'}}, dependent_library_name = 'guideline36'

    @classmethod
    def from_dict(
        cls, d: Dict[str, Any], dependent_library_name: str
    ) -> "_template_dependency":
        """Creates a py:class:`_template_dependency` from a dictionary.

        :param d: dictionary
        :type d: Dict[str, Any]
        :param dependent_library_name: library name
        :type dependent_library_name: str
        :return: the _template_dependency from the dict
        :rtype: _template_dependency
        """
>       template_name = d["template"]
E       KeyError: 'template'

buildingmotif/dataclasses/library.py:63: KeyError
```